neuro_san/client/direct_agent_session_factory.py

```diff
@@ -45,11 +45,7 @@ def __init__(self):
         """
         Constructor
         """
-        manifest_restorer = RegistryManifestRestorer()
-        self.manifest_networks: Dict[str, AgentNetwork] = manifest_restorer.restore()
-        self.network_storage = AgentNetworkStorage()
-        for agent_name, agent_network in self.manifest_networks.items():
-            self.network_storage.add_agent_network(agent_name, agent_network)
+        self.network_storage: AgentNetworkStorage = DirectAgentStorageUtil.create_network_storage()
```

Comment:
Use the common code factored out in DirectAgentStorageUtil to initialize from the manifest file.